src/controllers/trainee/routes.ts

```diff
@@ -1,19 +1,15 @@
 import{ Router } from 'express';
-
 import traineeControler from './controller';
 import validationHandler from '../../libs/validationHandler';
- 
-import config from '../trainee/validation';
-//import authmiddleware from '../../libs/routes/authmiddleware';
+import config from './validation';
+import authmiddleware from '../../libs/routes/authmiddleware';
 console.log(config);
-//console.log(authmiddleware);
+console.log(authmiddleware);
 const traineeRouter = Router();
 
 traineeRouter.route('/')
-.get( validationHandler ( config ) , traineeControler.get)
-.post(validationHandler ( config) , traineeControler.create)
-.put(validationHandler ( config) ,  traineeControler.update)
-.delete(validationHandler ( config) ,traineeControler.delete)
-//.get(authmiddleware('getUser','read'),validationHandler ( config ) , traineeControler.get)
-
+.get(authmiddleware('getUser','read'), validationHandler ( config ) , traineeControler.get)
```

Comment:
In validationHandler function, you are passing the whole config object pass route-specific config.
Like:
validationHandler ( config.get) for get route
validationHandler ( config.create) for  post route
validationHandler ( config.update) for  put route
validationHandler ( config.delete) for delete route